### ISSUE TITLE
java.lang.IllegalArgumentException: RGB is an unknown Mode

### ISSUE THREAD

**Issue by kevindesai777** [xrigau/androidthings-rgb-led#2]
Getting an IllegalArgumentException. 
Using dev preview 0.3

        try {
            mLedstrip = Ws2801.create(BoardDefaults.getSPIPort(), Ws2801.Mode.RGB);
            mLedstrip.write(new int[]{Color.parseColor("#0face0")});
            Log.d("LED", "Done!");
        } catch (IOException e) {
            Log.e("LED", "Error initializing LED strip", e);
        }

Code is from your demo app.

Logcat: 

```
05-14 00:02:05.530 4358-4358/com.app.led_androidthings E/AndroidRuntime: FATAL EXCEPTION: main
                                                                         Process: com.app.led_androidthings, PID: 4358
                                                                         java.lang.RuntimeException: Unable to start activity ComponentInfo{com.app.led_androidthings/com.app.led_androidthings.MainActivity}: java.lang.IllegalArgumentException: RGB is an unknown Mode
                                                                             at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2646)
                                                                             at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707)
                                                                             at android.app.ActivityThread.-wrap12(ActivityThread.java)
                                                                             at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460)
                                                                             at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                             at android.os.Looper.loop(Looper.java:154)
                                                                             at android.app.ActivityThread.main(ActivityThread.java:6077)
                                                                             at java.lang.reflect.Method.invoke(Native Method)
                                                                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865)
                                                                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755)
                                                                          Caused by: java.lang.IllegalArgumentException: RGB is an unknown Mode
                                                                             at com.xrigau.driver.ws2801.ColorUnpacker.getOrderedRgbBytes(ColorUnpacker.java:41)
                                                                             at com.xrigau.driver.ws2801.ColorUnpacker.unpack(ColorUnpacker.java:25)
                                                                             at com.xrigau.driver.ws2801.Ws2801.write(Ws2801.java:109)
                                                                             at com.app.led_androidthings.MainActivity.onCreate(MainActivity.java:29)
                                                                             at android.app.Activity.performCreate(Activity.java:6662)
                                                                             at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1118)
                                                                             at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2599)
                                                                             at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2707) 
                                                                             at android.app.ActivityThread.-wrap12(ActivityThread.java) 
                                                                             at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1460) 
                                                                             at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                             at android.os.Looper.loop(Looper.java:154) 
                                                                             at android.app.ActivityThread.main(ActivityThread.java:6077) 
                                                                             at java.lang.reflect.Method.invoke(Native Method) 
                                                                             at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:865) 
                                                                             at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:755) 
java.lang.IllegalArgumentException: RGB is an unknown Mode
```

**Comment by kevindesai777** [comment]
The error is in ColorUnpacker.java. Switch case does not list rgb.

**Comment by kevindesai777** [comment]
PR #3 raised to fix it

**Comment by kevindesai777** [comment]
Fixed with #3
